datalad/distribution/create_publication_target_sshwebserver.py

```diff
@@ -108,24 +109,28 @@ class CreatePublicationTargetSSHWebserver(Interface):
                 subdatasets of `dataset`""",),
         existing=Parameter(
             args=("--existing",),
-            constraints=EnsureChoice('skip', 'replace', 'raise'),
+            constraints=EnsureChoice('skip', 'replace', 'error'),
+            metavar='MODE',
             doc="""action to perform, if target directory exists already.
-                Dataset is skipped if `skip`. `replace` forces to (re-)init
-                git and to (re-)configure sibling `target`
-                (i.e. its URL(s)) in case it already exists.  `raise` just
-                raises an Exception""",),
+                Dataset is skipped if 'skip'. 'replace' forces to (re-)init
+                the dataset, and to (re-)configure the dataset sibling,
+                i.e. its URL(s), in case it already exists. 'error' causes an
+                exception to be raised.""",),
         shared=Parameter(
             args=("--shared",),
-            doc="""passed to git-init.  TODO: Figure out how to communicate what
-                this is about""",
+            metavar='false|true|umask|group|all|world|everybody|0xxx',
+            doc="""if given, configures the access permissions on the server
+            for multi-users (this could include access by a webserver!).
+            Possible values for this option are identical to those of
+            `git init --shared` and are described in its documentation.""",
             constraints=EnsureStr() | EnsureBool()),)
 
     @staticmethod
     @datasetmethod(name='create_publication_target_sshwebserver')
     def __call__(sshurl, target=None, target_dir=None,
                  target_url=None, target_pushurl=None,
                  dataset=None, recursive=False,
-                 existing='raise', shared=False):
+                 existing='error', shared=False):
```

Comment:
I would keep this change in mind to adjust my code to be "in-line"... I could argue that from Python API point of view 'raise' is more appropriate but since there it is exposed in user cmdline API, I guess indeed an `error` is the way